core/src/err/mod.rs

```diff
@@ -752,6 +752,20 @@ pub enum Error {
 	#[error("The password did not verify")]
 	InvalidPass,
 
+	/// The session has expired either because the token used
+	/// to establish it has expired or because an expiration
+	/// was explicitly defined when establishing it
+	#[error("The session has expired")]
+	ExpiredSession,
```

Comment:
These should be at the end of the enum, so that the sequence is the same across versions

Reply:
I was not aware of the fact the order was significant. I will move them to the end :+1:

Reply:
Done in [6604158](https://github.com/surrealdb/surrealdb/pull/3561/commits/6604158eb4e2eb583b75d1d23a592e4c500551f5).